Unable to start container, not enough diskspace

I set up your Photon Docker container on my Unraid server yesterday.
All went well, download speed was not the best but was going well.
Checking the logs this morning everything was fine; it finished after 6 hours.

Afterwards, I saw an update was available for the image (and also my server itself was waiting for one).
However, now I'm unable to start the container as I'm getting this error which is a bit weird.
`[ERROR] 2025-01-13 10:12:55 - Insufficient disk space. Required: 82987207173B , Available: 5524238336B`
The location I mounted is now using 320GB (total occupied space by Photon is 266 GB) and 179 GB free.
I think this should have enough space. Are Index Updates the same size as the full index?

I manually tagged the previous version (`0.6.1-rev.rev.4.2`) and restarted it.
Now the diskspace check does not happen but indeed checks for an index update.
Since I left it at the default setting (`SEQUENTIAL`) it started pulling the new (full) index.
As this is not what I was expecting I stopped the container, switched to `DISABLED` and started it again.
However now it said there was no index available and indeed when checking my storage, it's back down to 54 GB used.
Since the index seems already gone (this was not clear from the docs as it only said to stop Photon, not delete the files),
I went ahead and changed the tag to `latest` again but am still unable to start the container:
`[ERROR] 2025-01-13 10:35:31 - Insufficient disk space. Required: 82987207173B , Available: 5538000896B`

--- a/start-photon.sh
+++ b/start-photon.sh
@@ -64,8 +64,9 @@ check_disk_space() {
         return 1
     fi
     
-    # Check available space
-    available=$(df -B1 "$DATA_DIR" | awk 'NR==2 {print $4}')
+    # Check available space in photon_data directory
+    mkdir -p "$DATA_DIR/photon_data"
+    available=$(df -B1 "$DATA_DIR/photon_data" | awk 'NR==2 {print $4}')
     if [ "$available" -lt "$remote_size" ]; then
         log_error "Insufficient disk space. Required: ${remote_size}B , Available: ${available}B"
         return 1

